templates/store/snap-details/_channel_map.html

```diff
@@ -1,7 +1,7 @@
 <div id="js-channel-map" class="p-channel-map is-closed">
   <div id="channel-map-install" class="p-channel-map__tab">
     <div class="p-strip--light is-shallow">
-      <div class="row">
+      <div class="u-fixed-width">
```

Comment:
We use `u-fixed-width` a lot in this migration, and I'm not sure if it's good approach in long term.
Ideally this should be done with a `row` and `col-12`. I think Vanilla `u-fixed-width` is meant to be used when you don't want to or can't use the grid.
We can and should use the grid, but just use `u-fixed-width`, because class name replace is quicker fix. I already had issues with blindly use of `u-fixed-width` with floats in it.

Sure, we want to make this migration as quick as possible, and we could say 'we will update this later'. But practically speaking - will this 'later' be any time soon, or will we stay with `u-fixed-width` for a long time? 

@Lukewh Thoughts? As much as I hate adding these divs with `col-12` just to make things work I start to feel we should do it now properly.

Reply:
Unless we really create a post-migration task to get rid of `u-fixed-width` that we do soon after.

Reply:
I changed to `u-fixed-width` as per https://ubuntu.com/blog/vanilla-framework-2-0-upgrade-guide 2.4:
"Previously, a .row with a single col-12 or a col-12 on its own may have been used to display content in a fixed-width container. The nested solution adds unnecessary markup and a lone col-12 will no longer work.

A simple way to make an element fixed width, use the utility .u-fixed-width, which does not need columns."

Reply:
OK. If they recommend it, than it certainly should clear floats (like row did).

Reply:
Apparently using `u-clearfix` is the correct way to do it :man_shrugging: https://github.com/canonical-web-and-design/vanilla-framework/issues/2451